pyaccel/optics.py

```diff
@@ -349,20 +352,44 @@ def fitEllipse(x, y):
         res2 = _np.sqrt(up/down2)
```

Comment:
I think it makes code readability better if _up_ and _down2_ symbols are passed as arguments of _fitEllipse_. Besides, _up_ and _down_ names give no clue as to what these variables are.

Reply:
I see. these are local variables. I had misread the code. but the name suggestion persists.